src/main/scala/fi/oph/koski/valpas/ValpasKuntailmoitusService.scala

```diff
@@ -366,21 +372,32 @@ class ValpasKuntailmoitusService(
     tekijäOrganisaatio: Option[OrganisaatioWithOid],
     oppijanOppilaitokset: Set[ValpasOppilaitos.Oid]
   )(implicit session: ValpasSession): Either[HttpStatus, Set[OrganisaatioWithOid]] = {
-    tekijäOrganisaatio match {
-      // Jos organisaatio on annettu, ei palauteta toistaiseksi mitään muita vaihtoehtoja,
+    val sallitutRoolit = Set(
+      ValpasRooli.OPPILAITOS_HAKEUTUMINEN,
+      ValpasRooli.OPPILAITOS_SUORITTAMINEN,
```

Comment:
Minua vähän hämää, jos tässä käsitellään jo OPPILAITOS_SUORITTAMINEN, mutta ominaisuutta ei ole vielä toteutettu kokonaan. Mutta tietty jos tulee kohta uudessa commitissa, niin ei tästä mitään haittaa ole.

Reply:
Jäin eilisen keskustelun perusteella käsitykseen, että tämän voisi yhtä hyvin lisätä nyt. Voi tietty poistaakin.